### PR TITLE
added a signal for use of pages need republishing

### PR DESCRIPTION
**Pull request by fishibru**
Description of changes

- Signal added to show if a given page has been updated and needs republishing.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9271)

### CODE REVIEW

**Comment by kenotron** [comment]
@yumikohey, @ThomasMichon, @ahackettms - you seem to be the most active in this component. I'm not sure if this is an expected feature as the description is vague. Please take a look

**Comment by kenotron** [comment]
@fishibru - can you look at other PRs and provide a bit more info as to the nature of this issue? I simply do not have enough context to help review this.

File: packages/experiments/src/components/signals/Signals.scss [hunk 1]
```diff
@@ -176,4 +175,8 @@ i.emailed {
 
 .bookmarkFilled {
   color: $ms-color-neutralSecondary;
-}
\ No newline at end of file
+}
+
+.needsRepublishing {
+  color: #d29200;
```
Comment (KevinTCoughlin) [inline packages/experiments/src/components/signals/Signals.scss]: None of the other Signal colors are hard-coded, should this one be? I imagine we'd want to pull from our standard palette.

Comment (fishibru) [reply packages/experiments/src/components/signals/Signals.scss]: Good point. Updated file per comment.

**Comment by KevinTCoughlin** [comment]
@msft-github-bot require @ThomasMichon to approve before merge.

**Pull request merged**